site-src/guides/conformance-tests.md

```diff
@@ -0,0 +1,46 @@
+
+# Test Setup and Execution
+
+This document provides steps to run the Gateway API Inference Extension conformance tests.
+
+## Prerequisites
+
+1.  You need a Kubernetes cluster with [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) support.
+
+2.  Choose an Implementation -
+Install an [existing implementation](https://gateway-api-inference-extension.sigs.k8s.io/implementations/gateways/). For setup instructions, refer to the [The Quickstart Guide](https://gateway-api-inference-extension.sigs.k8s.io/guides/).  Alternatively run tests against your implementation after completing the [implementer's guide](https://gateway-api-inference-extension.sigs.k8s.io/guides/implementers/#implementers-guide).
+
+Note: Since the EPP (End Point Picker) takes the `InferencePool` name as an environment variable, each conformance test creates a corresponding EPP deployment for each `InferencePool` it defines. For conformance testing, the EPP is configured with the `HeaderBasedTestingFilter`. This is enabled by setting the `ENABLE_REQ_HEADER_BASED_SCHEDULER_FOR_TESTING=true` environment variable in the EPP deployment manifest. 
```

Comment:
@SinaChavoshi @zetxqx Is this actually needed if all our test cases are just telling the EPP which endpoint to select via header? Couldn't we just have a single base EPP that all InferencePools pointed to?

Reply:
Thank you @zetxqx, could i suggest we proceed with this PR and update it after https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/982 is merged ?